app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -12,17 +12,14 @@
   <%= link_to 'Export to EndNote', polymorphic_path([main_app, @presenter], format: 'endnote'), class: 'btn btn-default' %>
   <%= link_to 'Export to Zotero', sufia.static_path('zotero'), id: 'zoteroLink', name: 'zotero', class: 'btn btn-default' %>
   <%= link_to 'Export to Mendeley', sufia.static_path('mendeley'), id: 'mendeleyLink', name: 'mendeley', class: 'btn btn-default' %>
-
-  <% if @presenter.display_feature_link? %>
-    <%= link_to "Feature", sufia.featured_work_path(@presenter, format: :json),
-      data: {behavior: 'feature'}, id: 'featureLink', name: 'featureLink', class: 'btn btn-default' %>
+  <% if @presenter.respond_to?(:display_feature_link?) && @presenter.display_feature_link? %>
```

Comment:
@njaffer My quibble with checking the class name, rather than checking the behavior, is that we haven't said "FileSets will never be able to be highlighted," so why not build in the extra flexibility? I've seen over time that the assumptions I make don't necessarily jibe with those made by other Sufia adopters -- so if some adopter wants to make FileSets featurable, this lets them do that without requiring that they override one or more views.